pkg/mocks/user_proxy.go

```diff
@@ -0,0 +1,34 @@
+package mocks
+
+import (
+	"github.com/CESARBR/knot-babeltower/pkg/user/entities"
+	"github.com/stretchr/testify/mock"
+)
+
+// FakeUserProxy represents a mocking type for the user's proxy service
+type FakeUserProxy struct {
+	mock.Mock
+	Token string
+	Err   error
+	// Thing     *entities.Thing
```

Comment:
Oopss :grimacing: 

```suggestion
```

Reply:
OMG, haha. Tomorrow morning I'll remove it.

Reply:
Done.